03-specifying-sample-designs.Rmd

```diff
@@ -625,14 +612,14 @@ In specifying the design, the `variables` option was also used to include which
 
 There are three jackknife estimators implemented in {srvyr} - Jackknife 1 (JK1), Jackknife n (JKn), and Jackknife 2 (JK2). The JK1 method can be used for unstratified designs, and replicates are created by removing one PSU at a time so the number of replicates is the same as the number of PSUs. If there is no clustering, then the PSU is the ultimate sampling unit (e.g., unit). 
```

Comment:
the example at the end of this paragraph is "ultimate sampling unit (e.g., unit)".  Can we be more descriptive with this?  should we put "e.g., household" or "e.g., student"?  Feel like saying unit twice is not helping to clarify this.